src/iohandler/process_io_handler.cc

```diff
@@ -65,20 +65,11 @@ bool ProcListItem::abortOnDeath() const
 
 bool ProcessIOHandler::abort() const
 {
-    bool abort = false;
-
     if (procList.empty())
```

Comment:
This check is excessive. just `return std::any_of(...);`

Reply:
Removed.